pkgs/development/haskell-modules/configuration-common.nix

```
@@ -1254,4 +1254,7 @@ self: super: {
   # https://github.com/bitnomial/prometheus/issues/34
   prometheus = doJailbreak super.prometheus;
 
+  # Version 1.6.0 is no longer broken
```

```suggestion
  # The doctests in universum-1.5.0 are broken.  The doctests in versions of universum after
  # 1.5.0 should be fixed, so this should be able to be removed.
```

I fleshed out this comment a little bit to make it a little easier for the package maintainers to know when this line can be removed.